lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPluginUtils.java

```diff
@@ -42,18 +43,46 @@
 	private MavenPluginUtils() {
 		// Utility class, not meant to be instantiated
 	}
+	
+	public static String convertToBold(String message, boolean supportsMarkdown) {
+		StringBuilder bold = new StringBuilder();
+		if (supportsMarkdown) {
+			bold.append("**");
+		}
+		bold.append(message);
+		if (supportsMarkdown) {
+			bold.append("**");
+		}
+		
+		if (message.endsWith(" ")) {
+			bold.append(" ");
+		}
+		
+		return bold.toString();
+	}
 
-	public static MarkupContent getMarkupDescription(Parameter parameter) {
+	public static MarkupContent getMarkupDescription(Parameter parameter, boolean supportsMarkdown) {
 		String description = parameter.getDescription();
 		description = htmlXMLToMarkdown(description);
-		return new MarkupContent("markdown",
-				"**required:** " + parameter.getRequirement() + LINE_BREAK + "**Type:** " + parameter.getType() + LINE_BREAK
-						+ "Expression: " + parameter.getExpression() + LINE_BREAK + "Default Value: " + parameter.getDefaultValue()
-						+ LINE_BREAK + description);
+		
+		String markdownDescription = 
+				convertToBold("Required: ", supportsMarkdown) + parameter.isRequired() + LINE_BREAK + 
```

Comment:
If someone could verify that I am calling the correct function here. Before it for the "required" tag we would call parameter.getRequirement but comparing this to the name of the tag and similar code in the function below, I guess that it is supposed to be parameter.getRequired.

Reply:
both methods (parameter.isRequired() and parameter.getRequirement()) sound like they return the same value, maybe inspect their implementations to see if there are any differences?

Reply:
They return different things, one is just a boolean if the tag is "required" and the other is the component's requirements. I'm guessing the correct one here is isRequired.